CorsixTH/Src/sdl_core.cpp

```diff
@@ -29,6 +29,23 @@ SOFTWARE.
 #include "lua_sdl.h"
 #include "th_lua.h"
 
+
+GlobalTickCounter::GlobalTickCounter() {
+  counter = 0;
```

Comment:
Assign defaults in the initializer list not the body of the constructor.

`GlobalTickCounter::GlobalTickCounter() : counter(0) {}`